File: Array/codeforces_1879B.py

# Problem link ===>> https://codeforces.com/problemset/problem/1879/B

def solve():
    n = int(input())
    a = list(map(int, input().split()))
    b = list(map(int, input().split()))

    a.sort()
    b.sort()
    sum1 = n * b[0]
    sum2 = n * a[0]

    for i in range(n):
        sum1 += a[i]
        sum2 += b[i]

    print(min(sum1, sum2))


def main():
    t = int(input())
    for _ in range(t):
        solve()


if __name__ == "__main__":
    main()

File: Array/codeforces_1879B.cpp

// Problem link ===>> https://codeforces.com/problemset/problem/1879/B
// submission link ===>> https://codeforces.com/contest/1879/submission/241202860

#include <bits/stdc++.h>
using namespace std;
#define ll long long int

int main()
{

    int t;
    cin >> t;
    while (t--)
    {

        int n;
        cin >> n;

        vector<int> a;
        vector<int> b;

        for (int i = 0; i < n; i++)
        {
            int x;
            cin >> x;
            a.push_back(x);
        }
        for (int i = 0; i < n; i++)
        {
            int x;
            cin >> x;
            b.push_back(x);
        }
        sort(a.begin(), a.end());
        sort(b.begin(), b.end());
        long long h = 0;
        for (int i = 0; i < n; i++)
        {
            h += a[0] + b[i];
        }
        long long ans = 0;
        for (int i = 0; i < n; i++)
        {
            ans += b[0] + a[i];
        }
        cout << min(h, ans) << "\n";
    }

    return 0;
}